libethereum/ClientBase.cpp

```diff
@@ -377,6 +379,8 @@ Transaction ClientBase::transaction( h256 _blockHash, unsigned _i ) const {
         // allow invalid
         return Transaction( b[1][_i].data(), CheckTransaction::Cheap, true,
             EIP1559TransactionsPatch::isEnabledWhen(
+                blockInfo( numberFromHash( _blockHash ) - 1 ).timestamp() ),
```

Comment:
It's better to retrieve `blockInfo( numberFromHash( _blockHash ) - 1 ).timestamp()` into separate variable

Reply:
fixed